yt/python/yt/wrapper/auth_commands.py

```diff
@@ -19,19 +54,38 @@ def set_user_password(user, new_password, current_password=None,
         client=client)
 
 
+def set_user_password_interactive(user, client=None):
+    """Updates user password, reading it and optionally a current password
+    interactively and securely from console."""
+
+    current_password = maybe_request_password(user, client=client)
+    new_password = getpass("New password: ")
+    new_password_retyped = getpass("Retype new password: ")
+    if new_password != new_password_retyped:
+        raise YtError("Passwords do not match.")
+    return set_user_password(user, new_password, current_password=current_password,
+                             client=client)
+
+
 def issue_token(user, password=None,
                 client=None):
     """Issues a new token for user."""
     params = {"user": user}
     if password:
         params["password_sha256"] = encode_sha256(password)
-    params["output_format"] = "yson"
     return make_request(
         "issue_token",
         params=params,
         client=client)
 
 
+def issue_token_interactive(user, client=None):
+    """Issues a new token for user, reading password interactively and securely from console."""
+    password = maybe_request_password(user, client=client)
+    res = issue_token(user, password=password, client=client)
```

Comment:
Just `return`, additional variable is not needed

BTW: is this code written by some AI?

Reply:
Me + GH copilot in some variadic proportion